content/en/network_monitoring/devices/snmp_metrics.md

```diff
@@ -174,6 +174,84 @@ snmp_listener:
 
 **Note**: The Datadog Agent automatically configures the SNMP check with each of the IPs that are discovered. A discovered device is an IP that responds successfully when being polled using SNMP.
 
+### Ping
+
+When configured, the SNMP check can also send ICMP pings to your devices. This can be configured for individual as well as autodiscovered devices.
```

Comment:
```suggestion
When configured, the SNMP check can also send ICMP pings to your devices. This can be configured for individual as well as Autodiscovered devices.
```